numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -3363,3 +3439,20 @@ def get_version():
     Return the driver version as a tuple of (major, minor)
     """
     return driver.get_version()
+
+
+def inspect_obj_content(objpath: str):
```

Comment:
We need to be prepared for cuobjdump not to be available and handle that case. Similar handling for nvdisasm is at https://github.com/NVIDIA/numba-cuda/blob/main/numba_cuda/numba/cuda/codegen.py#L16

Reply:
Fixed in https://github.com/NVIDIA/numba-cuda/pull/62/commits/224eef4630adcfd5178598e3b8bc40f8423be1e5